bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -554,10 +555,12 @@ def from_api_repr(cls, resource):
         instance = cls(name)
         types = instance.struct_types
         for item in resource['parameterType']['structTypes']:
-            types[item['name']] = item['type']
+            types[item['name']] = item['type']['type']
```

Comment:
Yeah, for realz. https://cloud.google.com/bigquery/querying-data#bigquery-query-params-structs-api

It's because the type could also be an array or another struct, in which case it would need an `arrayType` or `structTypes`, too.